float32 nosewheelAngle
bool nosewheelShutdown
